nautobot/ipam/tests/test_views.py

```diff
@@ -328,6 +328,44 @@ def test_create_object_warnings(self):
                 strip_tags(content),
             )
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+    def test_prefix_ipaddresses_table_list_includes_child_ips(self):
+        ip_status = Status.objects.get_for_model(IPAddress).first()
+        instance = Prefix.objects.create(
+            prefix="5.5.10.0/23",
+            namespace=self.namespace,
+            type=PrefixTypeChoices.TYPE_NETWORK,
+            status=self.statuses[1],
+        )
+        Prefix.objects.create(
+            prefix="5.5.10.2/30",
+            namespace=self.namespace,
+            type=PrefixTypeChoices.TYPE_POOL,
+            status=self.statuses[1],
+        )
+        IPAddress.objects.create(
+            address="5.5.10.1/32",
+            status=ip_status,
+            namespace=self.namespace,
+        )
+        IPAddress.objects.create(
+            address="5.5.10.4/30",
+            status=ip_status,
+            namespace=self.namespace,
+        )
```

Comment:
```suggestion
        Prefix.objects.create(
            prefix="5.5.10.0/30",
            namespace=self.namespace,
            type=PrefixTypeChoices.TYPE_POOL,
            status=self.statuses[1],
        )
        IPAddress.objects.create(
            address="5.5.10.1/23",
            status=ip_status,
            namespace=self.namespace,
        )
        IPAddress.objects.create(
            address="5.5.10.4/23",
            status=ip_status,
            namespace=self.namespace,
        )
```

Just making the parent/child relationships a bit clearer.